src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1293,37 +1323,35 @@ internal void SendUndoToTutorial(TutorialState tutorial)
         tutorial.EditorUndoTutorial.EditorUndoButtonControl = undoButton;
     }
 
-    private static void SetOrganelleButtonStatus(Control organelleItem, bool nucleus)
+    private static void SetOrganelleButtonStatus(MicrobePartSelection organelleItem, bool nucleus)
     {
-        var button = organelleItem.GetNode<Button>("VBoxContainer/Button");
-
         if (organelleItem.Name == "nucleus")
```

Comment:
It would be pretty lovely to make this a property in the OrganelleDefinition loaded from json (I assume that it is pretty easy to access the organelle the organelleItem here is for).